docs/networking/deep-dive.md

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_position: 3
+sidebar_label: Deep Dive
+title: ""
+keywords:
+- Harvester
+- Networking
+- Topology
+---
+# Harvester Network Deep Dive
+
+The network topology below reveals how we implement the Harvester network.
+
+![](/img/v1.1/networking/topology.png)
+
+The diagram contains [the built-in cluster network mgmt](./clusternetwork.md/#built-in-cluster-network) and a [custom cluster network](./clusternetwork.md/#custom-cluster-network) called `oob`.
+
+As shown above, the Harvester network primarily focuses on layer 2. We leverage Linux network devices to construct traffic paths for the communication between VM to VM, VM to host, and VM to external network devices.
+
+The Harvester network is divided into three layers, including:
+
+- KubeVirt networking layer
+
+- Harvester networking layer
+
+- external networking layer
+
+## Kubevirt Networking
+
+The basic idea of KubeVirt is to put libvirt into the Kubernetes pod. The KubeVirt network is responsible for forwarding traffic between the pod and VM. 
+Please refer to the [KubeVirt official document](https://kubevirt.io/2018/KubeVirt-Network-Deep-Dive.html) for more details.
+
+## Harvester Networking
+
+Harvester networking is designed to forward the traffic from pods to the host network. It implements a management network, VLAN networks and untagged networks. We can refer to the last two networks as **bridge networks**, because bridge plays a vital role in their implementation.
```

Comment:
strictly speaking, `Harvester networking is designed to forward the traffic from pods to the host network.` is `Harvester networking is designed to forward the traffic from pods to the host network and vice versa.`

should we use `forward the traffic from .. to...` ? or `build/construct the network (path) between ... and ...` ?  network is not only `forward  the traffic`